lib/commands/permissions.js

```diff
@@ -101,7 +101,7 @@ commands.mobileResetPermission = async function mobileResetPermission (opts = {}
  * This method requires WIX applesimutils to be installed on the server host.
  *
  * @param {GetPermissionOptions} opts - Permission options.
- * @returns {string} Either 'yes', 'no' or 'unset'
+ * @returns {string} Either 'yes', 'no', 'unset' or 'limited'
```

Comment:
shall we also update README.md?

Reply:
yes, added 867b550a5a3abdef544658e53cd0759fe5abb53d